src/include/platform/internal/DeviceControlServer.h

```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *          Defines the Device Layer DeviceControlServer object.
+ */
+
+#pragma once
+
+#include <platform/internal/CHIPDeviceLayerInternal.h>
+
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+class DeviceControlServer final
+{
+public:
+    // ===== Members for internal use by other Device Layer components.
+
+    CHIP_ERROR HandleArmFailSafe(uint16_t expiryLengthSeconds);
```

Comment:
could we remove the Handle prefix?

ArmFailSafe seems clear enough. maybe even `ArmFailsSafeSec` to have the unit of time clearly defined.

Reply:
Acked